movieapp/src/containers/header/index.tsx

```diff
@@ -0,0 +1,15 @@
+import SearchForm from '../../components/searchform';
```

Comment:
wrote in private messages - nice to have absolute paths:  e.g.
```
import SearchForm from '@/components/searchform';
```

Reply:
fixed